lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -161,7 +162,7 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
       if verify_result
         start_normal_resource = Time.now.utc
         deploy_resources(resources, prune: prune, verify: true)
-        ::StatsD.measure('normal_resources.duration', StatsD.duration(start_normal_resource), tags: statsd_tags)
+        ::StatsD.distribution('normal_resources.duration', StatsD.duration(start_normal_resource), tags: statsd_tags)
```

Comment:
These first two were converted to use `measure_method` so they are already distributions on master. Please rebase. 🙏